components/automate-cli/cmd/chef-automate/version.go

```diff
@@ -165,27 +166,47 @@ func printUpgradeStatus() error {
 }
 
 func init() {
-	versionCmd.Flags().BoolVarP(&VersionCommandFlags.verbose, "verbose", "v", false, "Show additional version information")
```

Comment:
is this change is the part of some other PR ?

Reply:
Looks like it. This is not from this PR.